### PR TITLE
feat: new release notes cat for org & user mgmt

### PR DESCRIPTION
**Pull request by zuluecho9**
DOC-7255 for more info. A new 'Organization and user mgmt' release notes category.

### CODE REVIEW

File: src/content/docs/release-notes/org-user-mgmt-release-notes/org-users-21-09-01.mdx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+subject: Organization and user management
+releaseDate: '2021-09-01'
+version: '210901'
```
Comment (bradleycamacho) [inline src/content/docs/release-notes/org-user-mgmt-release-notes/org-users-21-09-01.mdx:4]: Just wanted to make sure this is the correct version number.

File: src/content/docs/release-notes/org-user-mgmt-release-notes/org-users-21-09-01.mdx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+subject: Organization and user management
+releaseDate: '2021-09-01'
```
Comment (bradleycamacho) [inline src/content/docs/release-notes/org-user-mgmt-release-notes/org-users-21-09-01.mdx:3]: Is this the correct date?

File: src/nav/accounts.yml [hunk 3]
```diff
@@ -15,6 +15,8 @@ pages:
             path: /docs/accounts/accounts-billing/account-setup/troubleshoot-new-relics-password-email-address-login-problems
           - title: Downgrade or cancel account
             path: /docs/accounts/accounts-billing/account-setup/downgradecancel-account
+          - title: Release notes
```
Comment (bradleycamacho) [inline src/nav/accounts.yml:18]: Is this the correct place in the taxonomy? Or should we move it to live as a direct child of "Accounts and billing".

This works, just wanted to make sure it was the correct place.

**Review by bradleycamacho — APPROVED** [review]
Looks good! Left a few comments, but this is solid.

File: src/content/docs/release-notes/org-user-mgmt-release-notes/org-users-21-09-01.mdx (hunk 1 above)
Comment (zuluecho9) [reply src/content/docs/release-notes/org-user-mgmt-release-notes/org-users-21-09-01.mdx:4]: yeah it doesn't really have a version number so i'm just using the date, which is how logs does it too.

File: src/content/docs/release-notes/org-user-mgmt-release-notes/org-users-21-09-01.mdx (hunk 2 above)
Comment (zuluecho9) [reply src/content/docs/release-notes/org-user-mgmt-release-notes/org-users-21-09-01.mdx:3]: yeah, been out a while.

**Pull request merged**